src/config.ts

```diff
@@ -16,5 +16,13 @@ export const RETRY_INTERVAL = 10000;
 export const POLLING_INTERVAL = 30000;
 export const CONFIRMATIONS = 1;
 export const CONFIRMATION_TIMEOUT = 10 * 60 * 1000; // 10 minutes
-export const GAS_MULTIPLIER = 160;
+export const GAS_LIMIT_MULTIPLIER = 160;
+export const GAS_PRICE_MULTIPLIER = 160;
 export const BALANCE_THRESHOLD = parseEther("0.05");
+export const GAS_STATION_API_ENABLED = true;
+export const GAS_STATION_API_CHAIN_ID = 1;
+export const GAS_STATION_API_URL =
+    "https://ethgasstation.info/json/ethgasAPI.json";
+export const GAS_STATION_API_KEY = undefined;
```

Comment:
How does this api key works? Is it required?

Reply:
[The documentation](https://docs.ethgasstation.info/#how-to-obtain-an-api-key) now requires the user to pass an api key, but it has been working for a long time without it and still is. It might change in the future tho, hence this config. We could later make the config editable through environment variables to let the user enable the gas station provider and provide an api key?